proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -43,9 +43,9 @@ enum ResourceType {
   RESOURCE_TYPE_USER = 1;
   RESOURCE_TYPE_ORGANIZATION = 2;
   RESOURCE_TYPE_ORGANIZATION_MEMBER = 3;
-  RESOURCE_TYPE_REPOSITORY = 4;
-  RESOURCE_TYPE_REPOSITORY_CONTRIBUTOR = 5;
-  RESOURCE_TYPE_REPOSITORY_COMMIT = 6;
+  RESOURCE_TYPE_MODULE = 4;
+  RESOURCE_TYPE_MODULE_CONTRIBUTOR = 5;
+  RESOURCE_TYPE_MODULE_COMMIT = 6;
```

Comment:
Commit is also changing to Snapshot ([PR](https://github.com/bufbuild/buf/pull/1448)). Are you planning a second PR for that change?

Reply:
Ah, missed that earlier. I think it's small enough to pull into this PR, no?

Reply:
just did 😈